game/Items.gd

```diff
@@ -8,7 +8,10 @@ enum LootType {
 	CURRENCY
 }
 enum ItemList {
-	CROWBAR,
+	MELEE_T0 = 0,
+	MELEE_T1 = 11,
```

Comment:
What are those numbers?

Reply:
They are the IDs and array positions for the items.

Reply:
Seems janky and error prone. But should be fine for now.

Reply:
You're right. I made a change to how we get items by ID.

It now searches the array instead of expecting the array positions and IDs to match up.

So, now the answer to the question is:
> They are the item IDs, and not the positions.

Reply:
Good call, btw.